tests/scripts/tests-setup.sh

```diff
@@ -3,12 +3,32 @@
 
 set -x
 
-if [ "$CONTAINER_USED" = "integration-test-snapshot" ]; then
-   export ARCH="--arch=$(uname -m)"
+# Install required dependencies only when asked
+if [ "$INSTALL_DEPS" == "yes" ]; then
+    dnf install \
+        podman \
+        python3-dasbus \
+        python3-podman \
+        python3-pytest \
+        python3-pytest-timeout \
+        -y
+fi
+
+# When building from packit testing farm we need to pass correct copr repo to the container
+BUILD_ARG=""
+if [ "$PACKIT_COPR_PROJECT" != "" ]; then
+    BUILD_ARG="--build-arg copr_repo=$PACKIT_COPR_PROJECT"
 fi
 
-podman build $ARCH -f ./containers/$CONTAINER_USED -t $BLUECHI_IMAGE_NAME .
+if [ "$CONTAINER_USED" = "integration-test-snapshot" ]; then
```

Comment:
Not really part of this PR, but is this check really needed? Can't we just always detect `arch`?

Reply:
Hmm, good catch, it's just a remain, which is no longer needed after https://github.com/eclipse-bluechi/bluechi/pull/441
I will remove it